data/data/aws/master/main.tf

```diff
@@ -1,5 +1,9 @@
 locals {
   arn = "aws"
+
+  // Because of the issue https://github.com/hashicorp/terraform/issues/12570, the consumers cannot use a dynamic list for count
+  // and therefore are force to implicitly assume that the list is of aws_lb_target_group_arns_length - 1, in case there is no api_external
```

Comment:
nit: 12570 was [closed with 0.12's better error message][1], and the actual support tracking has moved to [terraform#4149][2].  Dunno if it's worth updating your links?

[1]: https://github.com/hashicorp/terraform/issues/12570#issuecomment-512621787
[2]: https://github.com/hashicorp/terraform/issues/4149

Reply:
I think 12570 definitely has the correct context